a3p-integration/proposals/z:acceptance/localchain.test.js

```diff
@@ -27,7 +29,12 @@ test(`localchain passes tests`, async t => {
   const nodePath = 'test.localchain';
   const nodeValue = JSON.stringify({ success: true });
 
-  await waitForBlock(2); // enough time for core eval to execute ?
+  await retryUntilCondition(
```

Comment:
`retryUntilCondition` returns the produced value.
```suggestion
  const value = await retryUntilCondition(
```

Line 39 doesn't have to `await` it again.

Reply:
Requested change addressed